scalableminds/webknossos
# Script to apply mapping to volume tracing

### Steps to test:
- create a volume tracing, take note of an ID, download zip
- run `python3 tools/apply-mapping my-volumetracing.zip '{"7":"2"}'` to change cell ID 7 to 2
- (alternatively, place the json into a file and run it with `-f mapping.json`)
- upload the newly created `my-volumetracing_mapped.zip` to see if the change has succeeded

### Issues:
- fixes #3454 

------
